src/otx/core/types/label.py

```diff
@@ -41,9 +48,12 @@ def from_num_classes(cls, num_classes: int) -> LabelInfo:
                 label_groups=[["label_0", ...]]
             )
         """
+        if num_classes < 0:
```

Comment:
```suggestion
        if num_classes <= 0:
```
Is it intended to set `label_names` as [] if num_classes is 0 intead of returning `NullLabelInfo()`?

Reply:
Because this PR is huge and effective everywhere, I don't want to take charge of any possibility to break the test success if it is not a severe issue. Let's revisit it after this PR.